### DISCUSSION TITLE
Accept partial completions using ⌥→ while preserving normal behaviour when not showing completions?

### DISCUSSION THREAD

**Discussion by numist** [zed-industries/zed#20550] [Feature Requests]
The following does work for accepting partial completions:

``` json
{
  "context": "Editor",
  "bindings": {
    "alt-right": "editor::AcceptPartialInlineCompletion"
  }
}
```

But this (obviously) nerfs the default behaviour (moving the cursor by word, a macOS text editing mainstay). Following the example of https://github.com/zed-industries/zed/discussions/15845, I tried the following:

``` json
{
  "context": "Editor && showing_completions",
  "bindings": {
    "alt-right": "editor::AcceptPartialInlineCompletion"
  }
}
```

But this does not appear to work (on 0.160.7), is there something I'm missing?

**Comment by numist** [comment]
Oh had I read [further down the thread](https://github.com/zed-industries/zed/discussions/11474#discussioncomment-10949154) I might have noticed that this may have regressed recently?

Edit: Or that `inline_completion` is what I should have been using. GitHub search shows that `showing_completions` only exists in json configurations, which should probably be mopped up.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
